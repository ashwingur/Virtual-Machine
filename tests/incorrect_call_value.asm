FUNC LABEL 0
    PRINT VAL 5
    MOV REG 0 VAL 2
    CAL REG 0
    RET
FUNC LABEL 2
    PRINT VAL 1
    RET