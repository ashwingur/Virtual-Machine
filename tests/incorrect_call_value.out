5
CAL must be given a value type.
